src/utils/Mixins.scss

```diff
@@ -0,0 +1,21 @@
+@mixin center {
+  display: flex;
+  justify-content: center;
+}
+
+@mixin flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+@mixin onTablet {
+  @media (min-width: 744px) {
+    @content;
+  }
+}
+
+@mixin onDesktop {
```

Comment:
Consider using a more descriptive mixin name that describes what the mixin does. 🤔